release notes/v0.49.0.md

```diff
@@ -1,5 +1,6 @@
 k6 `v0.49.0` is here 🎉! This release includes:
 
+- a web-dashboard enabling users to track their load test results in real-time, directly in their web browser
```

Comment:
I'd suggest making this shorter, keeping in mind the context of the list
```suggestion
- a built-in web-dashboard for test run
```

Reply:
I think you're right, it's probably too long, I've given a shot to rephrase it to a shorter form indeed: "- a built-in web-dashboard displaying test results in real-time".

Reply:
It also works too :+1: